src/useTock.ts

```diff
@@ -62,7 +64,7 @@ function mapButton(button: any): Button {
   }
 }
 
-function mapCard(card: any): Card {
+function mapCard(card: any, recordResponseToLocaleSession: (card: any) => void): Card {
```

Comment:
Is this supposed to not be used?

Reply:
A leftover code of previous implementation